mm2src/mm2_main/tests/docker_tests/solana_tests.rs

```diff
@@ -131,14 +138,45 @@ fn test_disable_solana_platform_coin_with_tokens() {
     let res = block_on(disable_coin(&mm, "SOL-DEVNET", false));
     assert!(res.passivized);
 
-    // Try to disable ADEX-SOL-DEVNET and USDC-SOL-DEVNET
-    // This should work, because platform coin is still in the memory.
-    let res = block_on(disable_coin(&mm, "ADEX-SOL-DEVNET", false));
-    assert!(!res.passivized);
-    let res = block_on(disable_coin(&mm, "USDC-SOL-DEVNET", false));
-    assert!(!res.passivized);
-
     // Then try to force disable SOL-DEVNET platform coin.
     let res = block_on(disable_coin(&mm, "SOL-DEVNET", true));
     assert!(!res.passivized);
 }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_keypair_from_secp_dockerized() { solana_keypair_from_secp(); }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_transaction_simulations_dockerized() { solana_transaction_simulations(); }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_transaction_zero_balance_dockerized() { solana_transaction_zero_balance(); }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_transaction_simulations_not_enough_for_fees_dockerized() {
+    solana_transaction_simulations_not_enough_for_fees();
+}
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_transaction_simulations_max_dockerized() { solana_transaction_simulations_not_enough_for_fees(); }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_test_transactions_dockerized() { solana_test_transactions(); }
+
+// This test is just a unit test for brainstorming around tx_history for base_coin.
+#[test]
+#[ignore]
+#[cfg(not(target_arch = "wasm32"))]
+fn solana_test_tx_history_dockerized() { solana_test_transactions(); }
+
+#[test]
+fn solana_coin_send_and_refund_maker_payment_dockerized() { solana_coin_send_and_refund_maker_payment(); }
+
+#[test]
+fn solana_coin_send_and_spend_maker_payment_dockerized() { solana_coin_send_and_spend_maker_payment(); }
```

Comment:
Is it possible to move those tests here instead of wrapping them with this functions?

Reply:
Hello, @onur-ozkan I've moved the tests here, as you asked.